app/services/database_service.rb

```diff
@@ -0,0 +1,34 @@
+class DatabaseService
+    def self.get_url(url, params = nil)
+        response = conn.get(url)
+        JSON.parse(response.body, symbolize_names: true)
+    end
+    
+    def self.conn
+        Faraday.new(url: "http://localhost:3001")
+    end
+    
+    def self.recipe(meal)
+        # get_url("/api/v1/recipe", {meal: meal})
```

Comment:
Backend will need to huddle about how to use this search term, but great start! this is a BIG help